hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -253,6 +253,14 @@ public void persistToken(Token token) {
 			}
 			accountStore.persistAccount(token.getTreasury());
 		}
+		if (token.hasBurnedUniqueTokens()){
+			for (final UniqueToken uniqueToken : token.burnedUniqueTokens()) {
+				final var uniqueId = new MerkleUniqueTokenId(new EntityId(uniqueToken.getTokenId()), uniqueToken.getSerialNumber());
+				uniqueTokens.get().remove(uniqueId);
+			}
+			accountStore.persistAccount(token.getTreasury());
+		}
+		// if hasWiped ?
```

Comment:
nit: lets remove all comments related to wipe. We would like for it to be clean :)